### ISSUE TITLE
[ADE Research - Feedback] - Authenticated Experience Team: Profile (Checkbox saying “unavailable” even though it was focusable and could be checked)

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#139387]
> [!NOTE]
> - This is the child ticket of #`136864` - created by ADE to track **one** accessibility issue found, during a pilot session and/or user research session for the study.
> - This ticket will document the number of sessions in the study and/or pilot sessions that encountered this issue, along with the AT that was used.    

### Issue Identified

- **Name of Issue:** checkbox saying “unavailable” even though it was focusable and could be checked
- **Description of issue encountered:** The user was sent to the very top of the page when the checkbox was selected, while it was saving. The saving/update saved was not announced.  Focus did not stay in the checkbox 

<img width="1386" height="515" alt="Image" src="https://github.com/user-attachments/assets/7acbb1bb-8935-4d2a-904a-6500ed5492cc" />

#### Related Standards
- WCAG 1.3.1 – Info and Relationships
- WCAG 2.4.6 – Headings and Labels
- WCAG 1.4.4 – Resize Text

### Number of Issue Occurrences

 
| **Research** | **Date of Research** | **Browser** | **Assistive Tech** | **Device Type** | **P#** | **Shared in Feedback Channel** | 
|------------------------|------------------|------------|--------------------|-----------------|-------|-------|
| `User Session`                    |      `04`/`08`/2026       | `Chrome`    |        `JAWS`  |  `Desktop`           |       `P1`  | `Yes`    |        

#### Additional Issue Info: 
- **Is this issue a global issue?**
    - [ ] Yes
    - [x] No
    - [ ] Unknown at this time
 - **Is this issue occurring with multiple types of AT and/or on multiple devices, browsers, etc.?**
    - [ ] Yes - _if yes, update the table with your instance above._ 
    - [x] No
- **Is this issue a high-priority issue?**
    - [ ] Yes
    - [ ] No
    - [x] Unknown at this time
 - **Type of issue (select all that apply):**
    - [x] Accessibility (required)
    - [x] UX
    - [ ] UI
    - [ ] IA
    - [ ] Content
    - [ ] Other ____
